Add radius to prox parameters in here.js

mba7 opened the pull request:
in here.js
`prox: encodeURIComponent(location.lat) + ',' + encodeURIComponent(location.lng)`
give incomplete reversed address with only the city name 

adding raduis parameter will give the correct full address.
`prox: encodeURIComponent(location.lat) + ',' + encodeURIComponent(location.lng) + ',150'`

Commit:
Removed distributed leaflet version and use CDN version 0.6.4

Commit:
Rename Control.BingGeocoder* to Control.Geocoder

Commit:
Use all of the window for the map

Commit:
Control styling

Commit:
Show message when no matches are found instead of crashing

Commit:
Fix code after renaming from BingGeocoder to Geocoder

Commit:
Make geocoding provider easy to change by subclassing. Default to OSM/Nominatim provider and make Bing a subclass.

Commit:
Updated README.md and demo.

Commit:
Added demo link

Commit:
Style update.

Commit:
options.callback->options.onGeocodeResult; display marker with result's name

Commit:
Support for multiple results

Commit:
Refactor

Commit:
Added LICENSE according to https://github.com/sa3m/leaflet-control-bing-geocoder/issues/6

Commit:
Fixed title

Commit:
Added throbber

Commit:
Fixed issue where single geocode result crashed

Commit:
Broke out geocoding lookup to separate classes.

Commit:
Added class factories. Fixed missing commas.

Commit:
Improved docs and cleaned up API and code

Commit:
Fixed formatting snafu

Commit:
Module support and package.json

Commit:
Use class factory instead of new

Commit:
Added L.Control.Geocoder.RaveGeo. Fixed some issue with setting geocoder and removed unused key argument.

Commit:
Include NPM module status

perliedman committed:
jshint fixes

perliedman committed:
CSS lint fixes

perliedman committed:
Updated styling and HTML with inspiration from https://github.com/Esri/esri-leaflet-geocoder

perliedman committed:
Configurable expand mode, click is now default

perliedman committed:
Fixed throbber

perliedman committed:
Styling for alternatives

perliedman committed:
Keyboard selection

perliedman committed:
Remove alternatives when a selection is made

perliedman committed:
Event refactor

perliedman committed:
Latest fixes from gh-pages

perliedman committed:
Use escape to fix RaveGeo3's iso-8859 encoding

perliedman committed:
Updated docs, addressing #3

perliedman committed:
Select input's text when expanded. Closes #1

perliedman committed:
Collapse control when result is selected. Closes #2

perliedman committed:
Added optional icons for geocoding results, with option showResultIcons. Closes #4.

perliedman committed:
Updated selction color to match Leaflet's theme better

perliedman committed:
Added .jshintrc

perliedman committed:
Restructured layout, icon is a separate element and text can be selected without collapsing the control

perliedman committed:
Various CSS cleanup and fixes, many for touch. Fixes #7.

perliedman committed:
0.1.0

patrickarlt committed:
fix keyboard not showing in Chrome and Firefox and Android

perliedman committed:
Merge pull request #8 from patrickarlt/master

Keyboard does not appear on Chrome on Android

perliedman committed:
Added reverse geocoding support

perliedman committed:
Merge branch 'master' of github.com:perliedman/leaflet-control-geocoder

perliedman committed:
Fixed syntax error

perliedman committed:
Use L.DomUtil

perliedman committed:
0.2.0

perliedman committed:
More styling fixes for touch/mobile

perliedman committed:
Nominatim service default URL

perliedman committed:
Set _container before calling expand. Closes #9. Clear results when one is selected, if necessary.

perliedman committed:
Test for alternative control position

perliedman committed:
Add L.Control.Geocoder.MapQuest

perliedman committed:
Factory function

perliedman committed:
Use Leaflet 0.7.2

perliedman committed:
Better adaption to Leaflet 0.7's touch styling

perliedman committed:
Switch to 0.7.2 in all examples, removing special IE stylesheet

frodrigo committed:
Conform with other widget on touch device

perliedman committed:
Move icon for correct placement when error message is shown. Closes #11

perliedman committed:
0.2.1

Gia90 committed:
Update Control.Geocoder.js

perliedman committed:
Merge pull request #16 from Gia90/master

Update Control.Geocoder.js

perliedman committed:
Added support for specifying extra URL parameters for Nominatim. Fix empty response. Closes #15. Closes #13.

Commit:
Use protocol-relative urls when making requests

Commit:
Fix firefox border radius for form input, set vertical middle input position

perliedman committed:
Merge pull request #20 from tbicr/fix-styles

Fix firefox border radius for form input, set vertical middle input position

perliedman committed:
Re-added placeholder property

perliedman committed:
Don't use innerHTML for result strings, use proper text nodes to ensure correct HTML entity encoding. Fixes #19.

frodrigo committed:
Fix L.DomUtil.create with showResultIcons=true

perliedman committed:
Merge pull request #24 from frodrigo/master

Fix L.DomUtil.create with showResultIcons=true

perliedman committed:
Merge pull request #18 from jdebbink/workonhttps

Use protocol-relative urls when making requests

perliedman committed:
Removed window.location.protocol from service URLs, from accidentally merging 7e68366

perliedman committed:
Prevent default. Closes #25. Thanks to @Yermo.

Commit:
Mapbox support

Add support for Mapbox forward and reverse geocoding APIs

Commit:
Add Google support

Add Google forward and reverse geocoding support

Commit:
Use tab indentation

perliedman committed:
Merge pull request #29 from lucacorti/master

Mapbox support

perliedman committed:
Full list of providers

perliedman committed:
Include address details for Nominatim

perliedman committed:
Close control when escape is pressed

perliedman committed:
Support for HTML formatted addresses in Nominatim

perliedman committed:
Include notes on diversity in tech. Docs for new HTML formatting

perliedman committed:
Release v1.0.0

Commit:
Prevent Esc key from collapsing control when collapsed option is set to false. Close #36.

Commit:
Replace explicit check for single click with check for not double click, to fix problems with IE. Close #23. Close #35.

perliedman committed:
Example for `geocodingQueryParams`. Close #32.

tormi committed:
Update Control.Geocoder.js

Missing spaces in address popup

perliedman committed:
Merge pull request #44 from tormi/patch-1

Missing spaces in address popup

GuillaumeAmat committed:
Adds a bower.json file

perliedman committed:
Merge pull request #49 from GuillaumeAmat/master

Adds a bower.json file

junosuarez committed:
s/shrugging of/shrugging off/

perliedman committed:
Merge pull request #51 from jden/patch-1

s/shrugging of/shrugging off/

perliedman committed:
Add suggest method to Mapbox geocoder

perliedman committed:
1.1.0

jwerts committed:
added serviceUrl option to mapquest geocoder to support open api or future changes

perliedman committed:
Merge pull request #56 from jwerts/master

added serviceUrl option to mapquest geocoder to support open api or futu...

perliedman committed:
Add Photon geocoder support

ThiagoMVF committed:
Fixed the button size so it's the same one other leaflet controllers have and also fixed search box expansion causing the icon and overal component size to increase vertically

perliedman committed:
Merge pull request #63 from ThiagoMVF/tf-fix-size

Fix control size

Cyrille37 committed:
fixes #64

Cyrille37 committed:
fixes #66

Cyrille37 committed:
fixes #66 comment 2

Cyrille37 committed:
fixes #66 - oups! introduced an error

Cyrille37 committed:
fixes #66 - better order

Cyrille37 committed:
fixes #66 - missing a length check in Geocoder.Bing

davidpiesse committed:
Added support for what3words

Added new geocoder provider for what3words

perliedman committed:
Merge branch 'davidpiesse-patch-1'
Close #69

perliedman committed:
Merge pull request #67 from Cyrille37/pullrequest_for_perliedman

Fixed some bugs

Commit:
layout css simplified, previous had problems on Samsung tab default browser.
- transiotions removed, had artifacts when do not use display:none;

Commit:
removed commented style

Commit:
Merge branch 'master' of https://github.com/perliedman/leaflet-control-geocoder

perliedman committed:
Fixup indentation etc.

JonasKa committed:
Added "application/json" as Accept header param since all browsers (eg. Firefox) not add this by default

The consequence for this was my Here geocode service did return HTML by default, instead of JSON

perliedman committed:
Attach error element to container, not form

perliedman committed:
Use a elements for results; make both a and li clickable

perliedman committed:
Resets for a links in results; margin for error element

perliedman committed:
Merge branch 'vladimirbuskin-master'
Close #73.

perliedman committed:
Merge commit 'f0827ecc6172578e1f22834bdd65164636f26a81'
Close #75.

perliedman committed:
1.2.0

perliedman committed:
Move setRequestHeader to after opening request

perliedman committed:
1.2.1

Awem committed:
Improve empty link

perliedman committed:
Merge pull request #76 from AW-UC/improve-href

Improve empty link

Commit:
Clicking icon performs search when collapsed:false option is set

Commit:
Clicking icon performs search when collapsed:false option is set

Commit:
Add specificity to style of search icon

perliedman committed:
Merge pull request #81 from jdebbink/specificstyles

Add specificity to style of search icon

gregallensworth committed:
removed redundant clickHandler

The clickHandler was added to both the A and the underlying LI element. Clicking would propagate downward (even though it shouldn't) causing the result-click to happen twice. This can be demonstrated by a simple console.log() in your markGeocode() method.

perliedman committed:
Merge pull request #78 from jdebbink/iconclick

Clicking icon performs search when collapsed:false option is set

talos committed:
Options for Google Geocoder, address_components from its result

* Allow additional `options` for `Google` geocoder, which will be passed
  URL encoded to the `params` of the request.  This can be used to set
  boundaries, for example:

      ```
      var googleGeocoder = new L.Control.Geocoder.Google(null, {
        bounds: '40.49,-74.27|40.87,-73.68'
      }),
      ```

* Copy in the `address_components` details from Google's geocoder into
  results, as they contain useful data.  For example:

     ```
     geocoder.markGeocode = function (result) {
       // Congrats, you have access to houseNumber, street, and county in `result`
       var houseNumber = loc.address_components[0].short_name,
           street = loc.address_components[1].short_name,
           county = loc.address_components[4].short_name;
     };
     ```

perliedman committed:
Merge branch 'gregallensworth-patch-1'

perliedman committed:
Fix li items class set to 'undefined'

perliedman committed:
Fix result styling specificity to really override Leaflet's anchor styles

perliedman committed:
Add reverse geocoding for Photon

perliedman committed:
Merge branch 'master' of https://github.com/NYC-REIC/leaflet-control-geocoder into NYC-REIC-master

perliedman committed:
Update Google options handling and result parameter passing to be more in line with Nominatim's

perliedman committed:
Merge pull request #85 from talos

perliedman committed:
Break into smaller files; use browserify

perliedman committed:
Auto publish and download instructions

Commit:
1.3.0

wbg committed:
add Control.Geocoder.css to main

make wiredep happy

perliedman committed:
Merge pull request #90 from wbg/master

add Control.Geocoder.css to main

perliedman committed:
Use Leaflet 0.7.3, use built files from dist

perliedman committed:
Fix definition of Nominatim class

perliedman committed:
Fix missing dist folder in npm package

perliedman committed:
1.3.1

tomhughes committed:
Add support for Mapzen Search

tomhughes committed:
Add suggest support to the Mapzen geocoder

perliedman committed:
Merge pull request #93 from tomhughes/mapzen

Add support for Mapzen Search

tomhughes committed:
Fix the Mapzen factory function to propagate the API key

perliedman committed:
Merge pull request #94 from tomhughes/mapzen

Fix the Mapzen factory function to propagate the API key

Commit:
Remove trailing slash for Nominatim serviceUrl. Close #95.

Commit:
adding images to bower.json

perliedman committed:
Merge pull request #99 from restani/master

adding images to bower.json

Commit:
Use es3ify when building dist

Commit:
1.3.2

andreaswc committed:
Added HERE as a provider for geocoding and reverse geocoding

Commit:
Add hamlet's name in results

Commit:
Correctly append HTML after icon
Fix perliedman/leaflet-control-geocoder#103

perliedman committed:
Merge pull request #112 from ram-one/nominatim-fixes

Nominatim fixes

tsamaya committed:
fixes what3words link

perliedman committed:
Merge pull request #114 from tsamaya/patch-1

fixes what3words link

Commit:
Fix missing bind in Mapzen's suggest

Commit:
1.3.3

perliedman committed:
Try to fix publish to work for bower as well

perliedman committed:
Bump version 1.3.4

perliedman committed:
Run install instead of unknown script

perliedman committed:
Use GitHub's release page

kuzvac committed:
Add ArcGis geocoding service (#120)

* Add ArcGis geocoding service

* Add ArcGis geocoding service

* Rename acrgis to arcgis

* fix typo

* add missing ArcGis.class

* Rename arcgis to arcgis.js

* fix code

* Add ArcGis geocoding service

Commit:
Merge branch 'master' of git://github.com/andreaswc/leaflet-control-geocoder into andreaswc-master

Commit:
1.4.0

Commit:
Replace form tag with a normal div.

Close #86.

Commit:
Use HTTPS as default.

perliedman committed:
Cleanup control code

perliedman committed:
Update to Leaflet 0.7.7

perliedman committed:
Use cdnjs and hope for more stability

perliedman committed:
Avoid protocol relative URLs and prefer HTTPS

perliedman committed:
Add suggest functionality.

Close #53.

perliedman committed:
Refactor onAdd

perliedman committed:
Remove unused

perliedman committed:
Fire events to make customisation easier.

Close #42.

perliedman committed:
Always prevent default on return key.

Close #126.

perliedman committed:
Actually call suggest when in suggest mode

perliedman committed:
Fix parsing when result is a Point

davidpiesse committed:
Updated for v2 of w3w API (#131)

perliedman committed:
Fix selecting alt with mouse

perliedman committed:
Fix broken suggest/geocode logic; ignore old requests

perliedman committed:
Bump version to 1.5.0

perliedman committed:
Bump version to 1.5.1

perliedman committed:
Add link to DAWA geocoder

perliedman committed:
Badges badges badges badges

Commit:
Drop Leaflet dependency and use dist file as main

Commit:
Prevent control from collapsing on mousedown; close #142

Commit:
Update to use Leaflet 1.0.1

Commit:
Bump version to 1.5.2

Commit:
Fix so dist file can actually be used with browserify

Commit:
Bump to 1.5.3

Commit:
Remove Leaflet dep for bower; bump version to 1.5.4

gjaldon committed:
pass search data in callbacks (input and results) (#153)

perliedman commented:
@mba7 I think you might have submitted this PR to the wrong repository, it looks like the changes you made applies to leaflet-control-geocoder.

ghybs committed:
Doc(README): fix event.geocode.bbox property API (#161)

The `event.geocode` object that is passed to the `"markgeocode"` event listener does not have a `bounds` property, but a `bbox` one instead.

See for example the Nominatim adapter:
https://github.com/perliedman/leaflet-control-geocoder/blob/master/src/geocoders/nominatim.js#L53
(same for other adapters)

Corrected the README accordingly.

benscott committed:
Allow htmltemplate option for photon (#151)

quinnchr committed:
Adding ability to pass query options to the Mapbox API (#164)

* Adding ability to pass query options to the Mapbox API

* Woops, better make sure to check for empty proximity option first

* Moving proximity param conversion inside geocoding function and giving reverse its own set of options for Mapbox

Commit:
Fix paths to dist files in bower.json

Commit:
Bump version to 1.5.5

gregallensworth committed:
Mobile experience improvements (#169)

* Mobile experience improvements

* PR 169 indentation via tabs instead of spaces

* PR 169 'expand' default + added documentation

sedenardi committed:
inherit from 'L.Evented' instead of deprecated 'L.Mixin.Events' (#172)

Commit:
Bump version to 1.5.6

stevage committed:
Update README.md (#173)

* Update README.md

Remove link-breaking space

* Update README.md

Fix the URL while we're at it.
